pom.xml

```diff
@@ -145,24 +145,19 @@
       <version>1.3</version>
     </dependency>
     <dependency>
-      <groupId>com.google.apis</groupId>
-      <artifactId>google-api-services-container</artifactId>
-      <version>v1-rev57-1.25.0</version>
-    </dependency>
-    <dependency>
-      <groupId>com.google.apis</groupId>
-      <artifactId>google-api-services-cloudresourcemanager</artifactId>
-      <version>v1-rev535-1.25.0</version>
+      <groupId>com.google.graphite</groupId>
+      <artifactId>gcp-client</artifactId>
+      <version>1.0-SNAPSHOT</version>
```

Comment:
This needs to be changed before merging, but this version is used for testing and building locally.